### PR TITLE
fix a little typo in BalancedResourceAllocation

### PR DESCRIPTION
**Pull request by xuzhenglun**
Signed-off-by: Reficul <xuzhenglun@gmail.com>

**What this PR does / why we need it**:
fix a little typo in `BalancedResourceAllocation` of scheduler algorithm

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by xuzhenglun** [comment]
@k8s-ci-robot  I have signed the CLA.

**Comment by cjwagner** [comment]
/retest
(I'm just testing an infra update by checking that tests are triggered properly on non-member PRs)

**Review by bsalamat — COMMENTED** [review]
/ok-to-test

File: pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	balanceResourcePriority = &ResourceAllocationPriority{"BalanceResourceAllocation", balancedResourceScorer}
```
Comment (bsalamat) [inline pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: now that you have fixed this, could you please rename the variable to `balancedResourcePriority`?

**Comment by resouer** [comment]
Could you please add release note none to PR message?

File: pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go (hunk 1 above)
Comment (xuzhenglun) [reply pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: @bsalamat Done.

**Comment by xuzhenglun** [comment]
@resouer PR message has updated.

**Review by resouer — COMMENTED** [review]
/lgtm

**Pull request merged**